.buildkite/hooks/post-checkout

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+export APP_VERSION=$(cat package.json | python -c 'import json,sys;obj=json.load(sys.stdin);print obj["version"];')
+echo "+++ APP_VERSION: $APP_VERSION"
+
+# set TEST_ENVIRONMENT_NAME based on BUILDKITE_BRANCH
+case $BUILDKITE_BRANCH in
+develop)
+  export TEST_ENVIRONMENT_NAME=staging
```

Comment:
typo?  if expected add a comment why

Reply:
good catch! I was reading https://github.com/virtru/protect-and-track/blob/master/.buildkite/pipeline.yml#L99

and thought it should be `staging` for all non-master branches

But please let me if that is not the case...